libs/libarchfpga/src/physical_types.h

```diff
@@ -662,8 +681,25 @@ struct t_capacity_range {
     }
 };
 
-/** Describes the possible placeable blocks within a physical tile type.
- *  A sub tile adds flexibility in the tile composition description.
+/**
+ * @brief Describes the possible placeable blocks within a physical tile type.
+ *
+ * Heterogeneous blocks:
```

Comment:
Add a pointer to the heterogeneous tiles tutorial on the web (can just say look for it rather than giving a url), as that tutorial does help a lot in figuring this out.

Reply:
Done